### PR TITLE
[api-v1-event] add searchEvents

### PR DESCRIPTION
**Pull request by iskurenkov**

### CODE REVIEW

File: openapi.yaml [hunk 1]
```diff
@@ -1903,43 +1928,78 @@ components:
...
 
-    Activity:
+    EventSearchForm:
+      type: object
+      properties:
+        nextPage:
+          default: null
+          description: "Token to get the next page of results"
+          type: string
+        type:
```
Comment (evgenynacu) [inline openapi.yaml:1938]: I think we discussed not to use types like _FROM, _TO. But use only one type - TRANSFER

File: openapi.yaml [hunk 2]
```diff
@@ -1903,43 +1928,78 @@ components:
...
+          default: [TRANSFER_FROM, TRANSFER_TO, MINT, BURN, MAKE_BID, GET_BID, LIST, BUY, SELL, CANCEL_LIST, CANCEL_BID,
+                    AUCTION_BID, AUCTION_CREATED, AUCTION_CANCEL, AUCTION_FINISHED, AUCTION_STARTED, AUCTION_ENDED]
+          description: "Array of events"
+          type: array
+          items:
+            $ref: "#/components/schemas/UserEventType"
+        user:
```
Comment (evgenynacu) [inline openapi.yaml:1945]: the same, it's multiple, use s

File: openapi.yaml [hunk 3]
```diff
@@ -1903,43 +1928,78 @@ components:
...
+            $ref: "#/components/schemas/CollectionId"
+        item:
+          default: null
+          description: "Array of items"
+          type: array
+          items:
+            $ref: "#/components/schemas/ItemId"
+        createdAt:
+          $ref: "#/components/schemas/SearchDate"
+        lastUpdateAt:
```
Comment (evgenynacu) [inline openapi.yaml:1965]: I think this is not needed. It's our internal property

**Comment by evgenynacu** [comment]
Pls change back to Activity from Event

**Review by evgenynacu — CHANGES_REQUESTED** [review]

File: openapi.yaml (hunk 2 above)
Comment (iskurenkov) [reply openapi.yaml]: Fixed

File: openapi.yaml (hunk 1 above)
Comment (iskurenkov) [reply openapi.yaml]: Yes, I took this into account initially only in TransferActivity, updated it everywhere.

File: openapi.yaml (hunk 3 above)
Comment (iskurenkov) [reply openapi.yaml:1964]: Fixed

**Pull request merged**